From: Sr-0w <Sr-0w@users.noreply.github.com>
Date: Tue, 4 Aug 2026 18:20:00 +0200
Subject: [PATCH] arm64: dts: qcom: hotdog: enable WiFi after RMTFS

The isolated Hotdog RMTFS hardware gate boots the complete postmarketOS
system, assigns the 0xfc201000 shared-memory region, and brings the modem
remote processor to the running state. Enable the already-described WCN3990
platform now that its MPSS service dependency is available.
---
 arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts b/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts
--- a/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts
+++ b/arch/arm64/boot/dts/qcom/sm8150-oneplus-hotdog.dts
@@ -246,7 +246,7 @@
 };
 
 &wifi {
-	status = "disabled";
+	status = "okay";
 
 	vdd-0.8-cx-mx-supply = <&vreg_l1a_0p75>;
 	vdd-1.8-xo-supply = <&vreg_l7a_1p8>;
-- 
2.50.1
